src/pages/routes.test.ts

```diff
@@ -6,28 +6,13 @@ import { routes } from './RouteDefinitions'
 describe('Routes', () => {
   it('sitemap URLs should exist as Router paths', async () => {
     const pathNames: string[] = routes.map((routeDef) => routeDef.path)
-    await new Promise<boolean>((resolve, reject) => {
-      try {
-        fs.readFile('./public/sitemap.xml', 'utf8', async (err, data) => {
-          if (err) {
-            reject(err)
-          }
-          const sitemap = await parseStringPromise(data)
+    const contents = fs.readFileSync('./public/sitemap.xml', 'utf8')
+    const sitemap = await parseStringPromise(contents)
 
-          const sitemapPaths = sitemap.urlset.url.map((url: any) => new URL(url['$'].loc).pathname)
+    const sitemapPaths = sitemap.urlset.url.map((url: any) => new URL(url['$'].loc).pathname)
 
-          sitemapPaths.forEach((path: string) => {
-            expect(pathNames).toContain(path)
-            if (!pathNames.includes(path)) {
-              throw new Error(`${path} is missing from Routes`)
-            }
-          })
-
-          resolve(true)
-        })
-      } catch (err) {
-        reject(err)
-      }
+    sitemapPaths.forEach((path: string) => {
+      expect(pathNames).toContain(path)
```

Comment:
won't this err with not-found?

Reply:
no, i removed not-found from the sitemap